python/hail/plot/plots.py

```diff
@@ -146,21 +147,36 @@ def scatter(x, y, label=None, title=None, xlabel=None, ylabel=None, size=4, lege
         Size of markers in screen space units.
     legend : bool
         Whether or not to show the legend in the resulting figure.
+    collect_all : bool
+        Whether to collect all values or downsample before plotting.
+        Downsampling will only occur if x and y are both of type :class:`.Expression`.
+    n_divisions : int
+        Factor by which to downsample (default value = 500). A lower input results in fewer output datapoints.
     source_fields : Dict[str, List[Any]]
-        Extra fields for the ColumnDataSource of the plot.
+        Extra fields for the ColumnDataSource of the plot. Currently only used by calls from :meth:`manhattan`.
```

Comment:
I'd remove the comment it is only used by manhattan, because it is public, users could be using it, too!